### PR TITLE
feat(content): Add blog post on how to publish binaries on npm

### PR DESCRIPTION
**Pull request by lforst**
111 nice

### CODE REVIEW

File: data/blog/publishing-binaries-on-npm.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+tags: ['npm', 'binary', 'cli', 'web']
+draft: false
+summary: 'A comprehensive guide on how to publish binaries on npm without getting fired'
+images: [/images/publishing-binaries-on-npm/hero.jpeg]
+layout: PostLayout
+canonicalUrl: https://sentry.engineering/blog/publishing-binaries-on-npm
+authors: ['lucaforstner']
+---
+
+**This blog post is a comprehensive guide on how to distribute platform-specific binaries over npm.**
```
Comment (mydea) [inline data/blog/publishing-binaries-on-npm.md:13]: Hmm, I'd maybe rewrite this a bit to a proper introductory sentence or so 🤔 E.g.:

```suggestion
Did you ever have to publish binaries on npm? We do, and it turns out that doing so comes with a lot of potential pitfalls! But don't worry, this blog post can act as a comprehensive guide on how to distribute platform-specific binaries over npm.
```

File: data/blog/publishing-binaries-on-npm.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+   my-package
+   ├── package.json
+   ├── install.js
+   └── bin/
+       └── cli
+   ```
+
+Now we're done.
+Once `my-package` is published and installed, the binaries will be downloaded alongside, and you can either access them from your JS code with the `getBinaryPath()` function or you can directly invoke the binary executables from the command line.
+
```
Comment (Lms24) [inline data/blog/publishing-binaries-on-npm.md:352]: l/suggestion: Maybe add a "Wrapping Up"/conclusion paragraph to round off the post.

**Review by Lms24 — APPROVED** [review]
Great stuff!

File: data/blog/publishing-binaries-on-npm.md (hunk 1 above)
Comment (lforst) [reply data/blog/publishing-binaries-on-npm.md]: I settled for different wording but fleshed this out a bit. Thanks for the feedback!

File: data/blog/publishing-binaries-on-npm.md (hunk 2 above)
Comment (lforst) [reply data/blog/publishing-binaries-on-npm.md:353]: Yup that is a good idea. I rephrased this part a bit! Thanks! :)

**Pull request merged**